### DISCUSSION TITLE
Can you run Statamic alongside a regular Laravel app?

### DISCUSSION THREAD

**Discussion by pekka** [statamic/cms#8081] [General]
New-ish to Laravel and totally new to Statamic. I'm building a web app in Laravel and using Filament as the admin backend. 

Is it feasible to use Statamic for content management tasks within the same app? 

Most routes (regular page content) would have to be served by Statamic, while specific ones would have to be served by the app (and the /admin route by Filament).

Is this realistic, or will it lead to an unholy mess?

**Comment by jackmcdade** [comment]
I haven’t done it myself but know a number of people who do. I think it’s pretty straightforward!

**Comment by edalzell** [comment]
I've done it, it works great.
